x-pack/plugin/ccr/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/ccr/auto_follow.yml

```diff
@@ -25,6 +25,7 @@
         body:
           remote_cluster: local
           leader_index_patterns: ['logs-*']
+          leader_index_exclusion_patterns: ['logs-excluded']
```

Comment:
Can we add a test that it is still valid to not specify `leader_index_exclusion_patterns`?

Reply:
I think we should have two different tests and one should have a min version to be executed on, otherwise it might during bwc test.